src/renderers/shaders/ShaderChunk/premultiplied_alpha_fragment.glsl

```diff
@@ -0,0 +1,6 @@
+#ifdef PREMULTIPLIED_ALPHA
```

Comment:
#ifdef PREMULTIPLIED_ALPHA_BLENDING

    gl_FragColor.rgb *= gl_FragColor.a;

#endif

We do not want this to be confused with `renderer.premultipliedAlpha`.